flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/AutoScalerInfo.java

```diff
@@ -95,7 +95,7 @@ public SortedMap<Instant, CollectedMetrics> getMetricHistory() {
 
         try {
             return YAML_MAPPER.readValue(decompress(historyYaml), new TypeReference<>() {});
-        } catch (JsonProcessingException e) {
+        } catch (JacksonException e) {
```

Comment:
We could catch all `Exception`s maybe?

Reply:
The reason I changed the exception type is that this is the base Jackson exception which catches all Jackson related errors which we can't do anything about. Leaving this as-is for now.